src/net/sourceforge/kolmafia/textui/parsetree/ProxyRecordValue.java

```diff
@@ -1529,6 +1533,20 @@ public int get_turns_spent() {
           : 0;
     }
 
+    public int get_last_noncombat_turns_spent() {
+      if (this.content == null) {
+        return -1;
+      }
+
+      var id = ((KoLAdventure) this.content).getAdventureId();
+      // Using getString instead of getInteger for a default of -1.
```

Comment:
I don't understand this comment. Isn't it the case that `Preferences.getString` for a non-existent preference is `""`, and `StringUtilities.parseInt("")` is 0?